process_manager/tables.py

```diff
@@ -7,7 +7,17 @@
     "<a href=\"{% url 'process_manager:logs' record.uuid %}\">LOGS</a>"
 )
 
-header_checkbox_hyperscript = "on click set .row-checkbox.checked to my.checked"
+header_checkbox_hyperscript = """
+on click set .row-checkbox.checked to my.checked
+if my.checked
+    set #killButton.disabled to false
+    set #restartButton.disabled to false
+    set #flushButton.disabled to false
+else
+    set #killButton.disabled to true
+    set #restartButton.disabled to true
+    set #flushButton.disabled to true
```

Comment:
This has an unexpected behaviour - if there are no rows in the table, toggling the header checkbox still changes the state of the buttons. The state of the buttons should only be determined by whether there are any checked row checkboxes not by the state of the header checkbox. Can the `disableActionButton` behaviour just be extended to fire when the header checkbox is clicked (in addition to the row-checkboxes)?

Reply:
I tried to add it and it works, but it has some strange behaviour